sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -3543,7 +3534,7 @@ class AstBuilder extends SqlBaseBaseVisitor[AnyRef] with SQLConfHelper with Logg
    * }}}
    */
   override def visitReplaceTable(ctx: ReplaceTableContext): LogicalPlan = withOrigin(ctx) {
-    val table = visitReplaceTableHeader(ctx.replaceTableHeader)
+    val table = ctx.replaceTableHeader.multipartIdentifier.parts.asScala.map(_.getText).toSeq
```

Comment:
I think it's better to use `visitMultipartIdentifier(ctx.replaceTableHeader.multipartIdentifier)`

Reply:
updated